### ISSUE TITLE
If accessibility is disabled dialogs cannot be opened

### ISSUE THREAD

**Issue by chris-steele** [adaptlearning/adapt-contrib-core#616]
Observations:
 
- unopened dialogs are styled `display:none` by default
- [showModal ](https://github.com/adaptlearning/adapt-contrib-core/blob/master/js/a11y/popup.js#L100) is not called if accessibility has been disabled 
- problem began with core v6.60.3

**Comment by chris-steele** [comment]
@oliverfoster don't suppose you could have a look at this, could you?

**Comment by oliverfoster** [comment]
I don't understand what you mean by 'if accessibility is  disabled". Could you clarify plz?

**Comment by chris-steele** [comment]
Hi @oliverfoster, sorry for not being clearer, by disabling accessibility I meant setting the following:

```
"_accessibility": {
    "_isEnabled": false
```

ref: https://github.com/adaptlearning/adapt_framework/blob/master/src/course/config.json#L14

**Comment by oliverfoster** [comment]
Why are you setting it to _isEnabled: false? How did you find this bug? I need to know to tell you if the fix will work.

**Comment by oliverfoster** [comment]
As a warning to others:

Notify and Drawer now use [HTMLDialogElement.showModal](https://developer.mozilla.org/en-US/docs/Web/API/HTMLDialogElement/showModal) instead of the `[aria-hidden=true], [tabindex=-1]` method we've been using for years. 
 
Popups also now use CSS animations and positioning, rather than velocity animations and JavaScript calculations (except popup height, height is still derived and set in the JS).
 
The popups changed over to showModal and CSS in [this pr](https://github.com/adaptlearning/adapt-contrib-core/pull/601) and a bug was found when `config.json:_accessibility._isEnabled` was set to `false` resulting in [this pr](https://github.com/adaptlearning/adapt-contrib-core/pull/617). 
 
Note that the `_accessibility._isEnabled` property should never be set to `false` except in extreme circumstances (undefined) and its inclusion as an option in the AAT is suspect, should be eyeballed with extreme caution and the option should preferably be ignored as if its mere existence is an abomination to sanity.
 
It is worth noting that when upgrading a course or AAT to framework version [>=5.42.1](https://github.com/adaptlearning/adapt_framework/releases/tag/v5.42.1) (using core [>=6.60.3](https://github.com/adaptlearning/adapt-contrib-core/releases/tag/v6.60.3) ) that the above bug will appear only when setting `_accessibility._isEnabled` to `false`.
 
It is also worth noting that, if you have an existing theme, you will need to copy the default CSS popup animations and shadow animations from the corresponding [vanilla pr](https://github.com/adaptlearning/adapt-contrib-vanilla/pull/531/files). Without these defaults the Notify and Drawer popups will not animate.
